solr/CHANGES.txt

```diff
@@ -53,6 +53,10 @@ Other Changes
 * SOLR-13893: Remove support to read BlobRepository's max jar size from deprecated `runtme.lib.size` system property
   (Erick Erickson, Kesharee Nandan Vishwakarma, Munendra S N)
 
+* SOLR-12067: The default value of `CloudConfig.DEFAULT_AUTO_REPLICA_FAILOVER_WAIT_AFTER_EXPIRATION` which replaces the
```

Comment:
This statement is no longer true. Just mention `SOLR-12720: Remove support for autoReplicaFailoverWaitAfterExpiration` and add something to the upgrade notes saying that this is now removed and the way to change the auto add replica wait period is to change the `waitFor` attribute of `.auto_add_replicas` trigger.